scripts/zones/North_Gustaberg_[S]/mobs/Olgoi-Khorkhoi.lua

```diff
@@ -6,8 +6,17 @@ require("scripts/globals/hunts")
 -----------------------------------
 local entity = {}
 
+entity.onMobSpawn = function(mob)
+    mob:setRespawnTime(0, true)
```

Comment:
setRespawnTime only accepts one argument

Reply:
So the true doesn't need to be there. Alright.